docs/running-workflows/supported-systems/index.md

```diff
@@ -1,10 +1,11 @@
 # Supported systems
 
-| Engine           | Parametrised? | Parallel execution? | Partial execution? |
-| ---------------- | ------------- | ------------------- | ------------------ |
-| [CWL](cwl)       | yes           | yes                 | no (1)             |
-| [Serial](serial) | yes           | no                  | yes                |
-| [Yadage](yadage) | yes           | yes                 | no (1)             |
+| Engine                 | Parametrised? | Parallel execution? | Partial execution? |
+| ---------------------- | ------------- | ------------------- | ------------------ |
+| [CWL](cwl)             | yes           | yes                 | no (1)             |
+| [Serial](serial)       | yes           | no                  | yes                |
+| [Yadage](yadage)       | yes           | yes                 | no (1)             |
+| [Snakemake](snakemake) | yes           | yes                 | no (1)             |
```

Comment:
Alphabetically, Snakemake should be displayed before Yadage.

Reply:
Also, we could create one or two new columns expressing version concerns: e.g. as of which REANA version is the given system supported, and eventually also which CWL/Yadage version we are using.  Something like:

```
Yadage | 0.20.1 | 0.7.3 | yes | yes | no (1)
```